api/types/constants.go

```diff
@@ -567,6 +567,13 @@ const (
 	// downgraded before being returned to clients on older versions that do not
 	// support one or more features enabled in that resource.
 	TeleportDowngradedLabel = TeleportInternalLabelPrefix + "downgraded"
+
+	// TeleportManagedLabel identifies resources which are managed by Teleport. If needed, Teleport
+	// may modify these resources, but will not touch any user made modifications to these resources.
+	TeleportManagedLabel = TeleportInternalLabelPrefix + "managed"
```

Comment:
@tcsc also introduced a "system resource" label in https://github.com/gravitational/teleport/pull/27829. Let's please not have 2 different labels for the same thing. I'd probably vote for "teleport.internal/system: true".

Reply:
I added in the labels as we discussed. Let me know what you think.